### PR TITLE
000 repo setup

### PR DESCRIPTION
**Pull request by sammyperry101**
Setup repository for API including Github Action for Java with Maven and skeleton IntelliJ Project

### CODE REVIEW

File: src/main/java/org/kainos/ea/api/temp.txt
Comment (sammyperry101) [inline src/main/java/org/kainos/ea/api/temp.txt:1]: temp file added to ensure folder added to github

File: src/main/java/org/kainos/ea/cli/temp.txt
Comment (sammyperry101) [inline src/main/java/org/kainos/ea/cli/temp.txt:1]: temp file added to ensure folder added to github

File: src/main/java/org/kainos/ea/client/temp.txt
Comment (sammyperry101) [inline src/main/java/org/kainos/ea/client/temp.txt:1]: temp file added to ensure folder added to github

File: src/main/java/org/kainos/ea/core/temp.txt
Comment (sammyperry101) [inline src/main/java/org/kainos/ea/core/temp.txt:1]: temp file added to ensure folder added to github

File: src/main/java/org/kainos/ea/db/temp.txt
Comment (sammyperry101) [inline src/main/java/org/kainos/ea/db/temp.txt:1]: temp file added to ensure folder added to github

File: src/main/java/org/kainos/ea/health/temp.txt
Comment (sammyperry101) [inline src/main/java/org/kainos/ea/health/temp.txt:1]: temp file added to ensure folder added to github

File: src/main/java/org/kainos/ea/resources/temp.txt
Comment (sammyperry101) [inline src/main/java/org/kainos/ea/resources/temp.txt:1]: temp file added to ensure folder added to github

**Review by CallumE74 — APPROVED** [review]

**Review by TomBillingtonUK — APPROVED** [review]

**Pull request merged**